packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -240,10 +240,13 @@ export class MetamaskPage implements WalletPage {
       await this.navigate();
       await this.header.optionsMenuButton.click();
       await this.optionsMenu.menuAccountDetailsButton.click();
-      const address =
-        await this.popoverElements.accountDetailCopyAddressButton.textContent();
+      const address = await this.page
+        .locator(
+          '//div[@data-testid="address-copy-button-text"]/preceding-sibling::p',
```

Comment:
can you create a new locator "address" in that class?

Reply:
sure, my bad

Reply:
I'll stay with popover elements for now

Reply:
Fixed